lib/pharos/kube.rb

```diff
@@ -56,15 +58,15 @@ def self.config_exists?(host)
     # Shortcuts / compatibility:
```

Comment:
(These are not used much anymore, could be taken out)

Reply:
The `apply_stack` is still used in `ConfigureMetrics` 

FWIW the `apply_resource` is broken and never used, and the `prune_stack` is now unused, so those could be removed.

Reply:
Removed `apply_resource` & `prune_stack` (they are not used anymore).